dist/css/style.css

```diff
@@ -1,11 +1,491 @@
+@import url("https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;500;600;700;900&display=swap");
+
 * {
   box-sizing: border-box;
   padding: 0;
   margin: 0;
 }
 
-.body {
-  background-color: #000000;
-  height: 100vh;
-  width: 100vw;
-}/*# sourceMappingURL=style.css.map */
\ No newline at end of file
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: #e2e2e2;
+  font-family: Lato, sans-serif;
+}
+
+body button {
+  background-color: #f73a01;
+}
+
+body h2 {
+  font-weight: 600;
+}
+
+body h4 {
+  font-weight: 400;
+  color: #f73a01;
+}
+
+body h5 {
+  font-weight: 600;
+}
+
+.mobile {
+  width: 390px;
+}
+
+.mobile-hidden {
+  display: flex;
+}
+
+.desktop-hidden {
+  display: none !important;
+}
+
+header {
+  background: linear-gradient(rgb(211 211 211 / 90%), rgb(211 211 211 / 90%)), url("../images/photo1.jpg");
+  background-size: auto 100%;
+  background-repeat: no-repeat;
+}
+
+header nav {
+  height: 70px;
+  display: flex;
+  align-items: center;
+}
+
+header nav button {
+  height: 50px;
+}
+
+header .home-div {
+  padding: 32px 24px 10px;
+  padding-bottom: 20px;
+}
+
+header .home-div .date {
+  margin-top: 30px;
+}
+
+header .home-div hr {
+  height: 10px;
+}
+
+header .home-div-about {
+  padding: 15px 24px;
+  text-align: center;
+}
+
+header .email-about {
+  margin-top: 30px;
+}
+
+header span {
+  text-decoration: underline;
+  font-weight: 600;
+}
+
+header .about-card {
+  margin-top: 38px;
+  border: 1px solid rgb(70 70 70 / 30%);
+  background-color: #e2e2e2;
+  padding: 20px 15px 10px;
+  text-align: center;
+  border-radius: 4px;
+}
+
+section .program {
+  width: 98%;
+  margin-left: auto;
+  margin-right: auto;
+  border-radius: 5px;
+  background-color: rgb(255 255 255 / 4.8%);
+}
+
+section .program h5 {
+  color: #f73a01;
+}
```

Comment:
- [ ] @SamTush great work implementing the changes requested, however, you forgot to change the text  color for the header, you currently are using color: #f73a01; instead of #EC5242 recommended in the design.